src/db/migrations/20250513034953_job_type_updates/migration.sql

```diff
@@ -0,0 +1,28 @@
+-- Custom migration to handle CRITERIA_POLYGONS removal
+BEGIN;
+
+-- Delete JobResults linked to Jobs with type CRITERIA_POLYGONS
+DELETE FROM "JobResult" 
+WHERE "job_id" IN (SELECT "id" FROM "Job" WHERE "type" = 'CRITERIA_POLYGONS');
+
+-- Delete JobAssignments linked to Jobs with type CRITERIA_POLYGONS
+DELETE FROM "JobAssignment" 
+WHERE "job_id" IN (SELECT "id" FROM "Job" WHERE "type" = 'CRITERIA_POLYGONS');
+
+-- Delete JobRequests linked to Jobs with type CRITERIA_POLYGONS
+DELETE FROM "JobRequest" 
+WHERE "type" = 'CRITERIA_POLYGONS';
+
+-- Delete Jobs with type CRITERIA_POLYGONS
+DELETE FROM "Job" 
+WHERE "type" = 'CRITERIA_POLYGONS';
+
+-- Now we can safely modify the enum
+CREATE TYPE "JobType_new" AS ENUM ('TEST', 'SUITABILITY_ASSESSMENT');
```

Comment:
awkward. wondering if we should make job type a table/relation instead of enum

Reply:
That's the auto generated Prisma migration!

Reply:
The deletion part at the top was my bit